### PR TITLE
[DOCS-4611] Add Linux App Services

### PR DESCRIPTION
**Pull request by Danny-Driscoll**
Adding a new section for Linux Azure App Services setup

### What does this PR do?
Add a new index page for Azure App Services, and add a new Linux App Services documentation page

### Motivation
We are beginning a public Beta for Linux Azure App Services APM

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by kayayarai** [comment]
Opened https://datadoghq.atlassian.net/browse/DOCS-4611 to track docs team review

File: content/en/serverless/azure_app_services/azure_app_services_linux.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+The solution leverages the Startup Command setting, and Application Settings for Linux Azure App services to instrument the application and manage the configuration. 
+
+## Setup
+### Configuration
+To instrument your application, begin by adding the following key value pairs as Application Settings. Note that the application will re-start when new settings are saved. 
+
+- `DD_API_KEY` is your Datadog API key
+- `DD_SITE` is the Datadog site [parameter][2]) (defaults to datadoghq.com)
+- `DD_SERVICE` is the service name used for this program. Defaults to the name field value in package.json.
+- `DD_START_APP` is the command used to start your application. For example, `node ./bin/www`
```
Comment (cswatt) [inline content/en/serverless/azure_app_services/azure_app_services_linux.md:20]:
```suggestion
- `DD_API_KEY` is your Datadog API key.
- `DD_SITE` is the Datadog [site parameter][2]. This defaults to `datadoghq.com`.
- `DD_SERVICE` is the service name used for this program. Defaults to the name field value in `package.json`.
- `DD_START_APP` is the command used to start your application. For example, `node ./bin/www`.
```

File: content/en/serverless/azure_app_services/azure_app_services_linux.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+![storms-nodejs-example - Microsoft Azure 2022-11-30 at 4 15 47 PM](https://p-qkfgo2.t2.n0.cdn.getcloudapp.com/items/YEuD88kN/57eceb6b-dd34-4d5f-a7ea-a8fcc2ec77ba.jpg?source=viewer&v=505cc168a458a4ec84b7d6a903f78493)
+
+### Startup Command
```
Comment (cswatt) [inline content/en/serverless/azure_app_services/azure_app_services_linux.md:24]:
```suggestion
### Startup command
```

File: content/en/serverless/azure_app_services/azure_app_services_linux.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Next, add the following to the startup command field under General Settings.
+
+
+      curl -s https://raw.githubusercontent.com/DataDog/datadog-aas-extension/linux-v0.1.4-beta/linux/datadog_wrapper | bash
```
Comment (cswatt) [inline content/en/serverless/azure_app_services/azure_app_services_linux.md:28]:
```suggestion
`curl -s https://raw.githubusercontent.com/DataDog/datadog-aas-extension/linux-v0.1.4-beta/linux/datadog_wrapper | bash`
```

File: content/en/serverless/azure_app_services/azure_app_services_linux.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+![storms-nodejs-example - Microsoft Azure 2022-11-30 at 4 15 26 PM](https://p-qkfgo2.t2.n0.cdn.getcloudapp.com/items/P8uNWWQ6/02c4f33f-f4d9-42b3-b746-3d5c9d62a8f3.jpg?source=viewer&v=3db9f9bba7f342e88c43da5aed1218fd)
+
+
+### Viewing traces
+
+After the application has been instrumented and restarted successfully, traces will be available from the application within. These can be accessed from  the [Azure Serverless view][3], or in the [APM Service page][4] of your Datadog app.
```
Comment (cswatt) [inline content/en/serverless/azure_app_services/azure_app_services_linux.md:35]:
```suggestion
After the application is instrumented and successfully restarted, traces are available from the application within. You can access these in the [Azure Serverless view][3], or in the [APM Service page][4] of Datadog.
```

File: content/en/serverless/azure_app_services/azure_app_services_linux.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+### Custom Metrics
```
Comment (cswatt) [inline content/en/serverless/azure_app_services/azure_app_services_linux.md:37]:
```suggestion
### Custom metrics
```

**Review by cswatt — CHANGES_REQUESTED** [review]

**Review by cswatt — APPROVED** [review]

**Comment by jcstorms1** [comment]
@cswatt @Danny-Driscoll  I just released the first version of this. The only changes we need are the links and snapshots should be updated. These can be found in the new Readme https://github.com/DataDog/datadog-aas-linux/#readme

**Comment by cswatt** [comment]
new pr: https://github.com/DataDog/documentation/pull/16384

**Pull request closed without merging**